ci-operator/step-registry/dpu-operator/e2e-tests-workflow/tests/dpu-operator-e2e-tests-workflow-tests-commands.sh

```diff
@@ -1,79 +1,96 @@
 #!/bin/bash
 
+echoerr() { echo "$@" 1>&2; }
+
+trigger_build() {
+	local pullnumber=$1
```

Comment:
all these variables are non changing... we only need to define them once

Reply:
These are arguments to functions. The old code was using globals.

Reply:
right... my argument is that there is a difference between global variables and static variables. None of these variables are ever changing (they are static). eg PULL_NUMBER, token, endpoint, and test name will never be expected to change throughout the program. Hence the suggestion to define once.